messagebird/call_data.py

```diff
@@ -0,0 +1,37 @@
+from messagebird.base import Base
+
+class CallData(Base):
+    
+    def __init__(self):
```

Comment:
You also have `webhook` which is composed of a `token` and a `url`

Reply:
ok, i will add that

Reply:
@dysosmus just added webhook to Call class